src/openfl/_internal/renderer/opengl/GLMaskManager.hx

```diff
@@ -24,7 +24,7 @@ import openfl.utils.ByteArray;
 class GLMaskManager extends AbstractMaskManager {
 	
 	
-	@:allow(openfl._internal.renderer.opengl) private static var maskShader = new GLMaskShader ();
+	public var maskShader(default,null) = new GLMaskShader ();
```

Comment:
Was it needed to recreate the mask shader on context loss, while otherwise as a `static` it would just exist once?

Reply:
Yes, because `Shader` instances reference the gl context once initialized. We could un-init it, but I think simply having a new instance is cleaner. Also this is what's done to the default shader.